pkg/evs/nodeserver.go

```diff
@@ -106,6 +106,16 @@ func (ns *nodeServer) NodeStageVolume(_ context.Context, req *csi.NodeStageVolum
 	return &csi.NodeStageVolumeResponse{}, nil
 }
 
+func getDevicePath(id string, mount mounts.IMount) (string, error) {
+	var devicePath string
+	devicePath, _ = mount.GetDevicePath(id)
```

Comment:
err message shoule be retained。
err message should be returned if err != nil

Reply:
yes, err shouldn't be ignored, but it's also error as long as devicePath is empty.